### DISCUSSION TITLE
did anyone get this issue Method App\Http\Livewire\CustomSteps\confirmorder::state does not exist.

### DISCUSSION THREAD

**Discussion by Programmers12** [spatie/laravel-livewire-wizard#16] [Q&A] [answered]
public function render()

    {

        return view('livewire.steps.confirm-order', [

            **_**'amount' => $this->state()->amount(),**_**//this the erreur

            'address' => $this->state()->deliveryAddress(),

        ]);

    }

**Comment by freekmurze** [comment, marked as answer]
Make sure you are using the latest version, and also make sure you extend our step component.
